### PR TITLE
Add integration and acceptance tests

### PR DESCRIPTION
**Pull request by voscob**
### Description

Integration and acceptance tests were added.

Fixes # (issue)

### Quick checks:

- [x] There is no other [pull request](https://github.com/conduitio-labs/conduit-connector-cosmos-nosql/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: acceptance_test.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+	"github.com/conduitio-labs/conduit-connector-cosmos-nosql/source"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/matryer/is"
+	"go.uber.org/goleak"
+)
+
+type driver struct {
+	sdk.ConfigurableAcceptanceTestDriver
+
+	orderInt int64
```
Comment (maksenius) [inline acceptance_test.go:37]: maybe we can rename it to `orderID`

File: acceptance_test.go [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+	sdk.AcceptanceTest(t, &driver{
+		ConfigurableAcceptanceTestDriver: sdk.ConfigurableAcceptanceTestDriver{
+			Config: sdk.ConfigurableAcceptanceTestDriverConfig{
+				Connector:         Connector,
+				SourceConfig:      cfg,
+				DestinationConfig: cfg,
+				BeforeTest:        beforeTest(cfg),
+				AfterTest:         afterTest(cfg),
+				GoleakOptions: []goleak.Option{
+					goleak.IgnoreTopFunction("internal/poll.runtime_pollWait"),
```
Comment (maksenius) [inline acceptance_test.go:91]: Can we fix this leak?

File: acceptance_test.go (hunk 1 above)
Comment (voscob) [reply acceptance_test.go:37]: I've renamed it to `i`.

File: acceptance_test.go (hunk 2 above)
Comment (voscob) [reply acceptance_test.go:92]: I've added the comment.

**Review by maksenius — APPROVED** [review]

**Pull request merged**